### ISSUE TITLE
<b:isolate/> by default

### ISSUE THREAD

**Issue by istrel** [basisjs/basisjs#146]
For now, if you want to use CSS modules, you need to specify `<b:isolate/>` everywhere and it's so easy to miss it in some files. 
Moreover, if you need to override some isolated component, you should specify `<b:isolate/>` again despite the fact, that you may not change any styles.

So it seems to be a good solution to add ability to specify `isolateByDefault` in `basisJsConfig` and add ability to populate CSS via `<b:noisolate/>` declaration

**Comment by lahmatiy** [comment]
Yep, we discussed this enhancement times ago. It's nice to have.
Do you want work on PR?

**Comment by lahmatiy** [comment]
But I think it would be better to turn on `<b:isolate>` by default in `2.0`. An option is not a good solution if we think about external components in future.
So we need implement `<b:noisolate/>` instruction - it's quite easy. `<b:isolate>` continue to work as is and would be used to customize prefix if needed. The hardest place is tests that should be changed to new behaviour.

**Comment by istrel** [comment]
Yes, I want
